.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Ticket Number
+<!-- Add the number from the Jira board -->
+UCD-
+
+## Description of change
+<!-- Please describe the change -->
+
+## Checklist
+<!-- Put an `x` in all the boxes that apply: -->
+- [ ] Requires infrastructure changes
+- [ ] I have updated the documentation accordingly
+- [ ] I have merged commits to avoid fixing things in this PR
+- [ ] New features have tests
+- [ ] All new and existing tests pass
```

Comment:
For L13-14 we could negate the need for these items on the checklist by:

1. Ensuring the build fails if cypress or jest test fail (I think this is as-is anyway)
2. We could set a coverage threshold to catch new features needing tests 

Open to discussion on either point

Reply:
I like the idea of adding coverage thresholds - currently, it's all to 100% in the appeals-api, but doesn't have "all files set" so it's only 100% of all files that have 1 or more tests

I think I'll get rid of line 14. We can get rid of line 13 in future (do you want to lead that process?)

Reply:
Yes, I will look into it. Drives me nuts when stuff doesn't have tests tbh

Reply:
I think I :heart: you